pyart/retrieve/echo_class.py

```diff
@@ -997,75 +997,75 @@ def conv_strat_raut(
+
+    This function uses à trous wavelet transform (ATWT) for multiresolution (scale) analysis of radar field,
+    focusing on precipitation structure over reflectivity thresholds for classification (Raut et al 2008, 2020).
+
+    Parameters
+    ----------
+    grid : Grid
+        Grid object containing radar data.
+    refl_field : str
+        Field name for reflectivity data in the Py-ART grid object.
+    zr_a : float, optional
+        Coefficient 'a' in the Z-R relationship Z = a*R^b for reflectivity to rain rate conversion.
+        The algorithm is not sensitive to precise values of 'zr_a' and 'zr_b'; however,
+        they must be adjusted based on the type of radar used.
+        Default is 200.
+    zr_b : float, optional
+        Coefficient 'b' in the Z-R relationship Z = a*R^b. Default is 1.6.
+    core_wt_threshold : float, optional
+        Threshold for wavelet components to separate convective cores from mix-intermediate type.
+        Default is 5. Recommended values are between 4 and 6.
+    conv_wt_threshold : float, optional
+        Threshold for significant wavelet components to separate all convection from stratiform.
+        Default is 1.5. Recommended values are between 1 and 2.
+    conv_scale_km : float, optional
+        Approximate scale break (in km) between convective and stratiform scales.
+        Scale break may vary over different regions and seasons
+        (Refere to Raut et al 2018 for more discussion on scale-breaks). Note that the
+        algorithm is insensitive to small variations in the scale break due to the
+        dyadic nature of the scaling. The actual scale break used in the calculation of wavelets
+        is returned in the output dictionary by parameter `scale_break_used`.
+        Default is 25 km. Recommended values are between 16 and 32 km.
+    min_reflectivity : float, optional
+        Minimum reflectivity threshold. Reflectivities below this value are not classified.
+        Default is 5 dBZ. This value must be greater than or equal to '0'.
+    conv_min_refl : float, optional
+        Reflectivity values lower than this threshold will be always considered as non-convective.
+        Default is 25 dBZ. Recommended values are between 25 and 30 dBZ.
+    conv_core_threshold : float, optional
+        Reflectivities above this threshold are classified as convective cores if wavelet components are significant (See: conv_wt_threshold).
+        Default is 42 dBZ.
+        Recommended value must be is greater than or equal to 40 dBZ. The algorithm is not sensitive to this value.
+    override_checks : bool, optional
+        If set to True, the function will bypass the sanity checks for above parameter values.
+        This allows the user to use custom values for parameters, even if they fall outside
+        the recommended ranges. The default is False.
+
+    Returns
+-------
```

Comment:
The lines under Returns doesn't seem to be lined up still

Reply:
Sorry for this one docstring giving issues. I don't know why? but `Black` seams to `tab` that one docstring. Corrected now. Please check. Thanks for all the feedbacks.